front/ui/ui-icons/src/components/_template.tsx

```diff
@@ -33,7 +33,7 @@ const IconReplaceName: IconReplaceNameIcon = ({
         height={currentSize}
         fill={iconColor || 'currentColor'}
         viewBox={`0 0 ${currentSize} ${currentSize}`}
-        dangerouslySetInnerHTML={{ __html: data }}
+        dangerouslySetInnerHTML={useMemo(() => ({ __html: data }), [data])}
```

Comment:
We have another one of these in `front/src/applications/stdcm/components/StdcmHelpModule/SectionContentManager.tsx`. Do we need to update that one as well?

Reply:
I couldn't find out why this is happening, but I don't have more time to investigate. Let's leave it at that!